src/storaged/src/main.rs

```diff
@@ -98,6 +98,10 @@ struct Args {
     /// The address of the HTTP profiling UI.
     #[clap(long, value_name = "HOST:PORT")]
     http_console_addr: Option<String>,
+
+    /// Add the process name to the tracing logs
+    #[clap(long, hide = true)]
+    log_process_name: bool,
```

Comment:
I like to bias against optionality whenever possible. Can we mandate this is `true`?